cadc-vosi/build.gradle

```diff
@@ -16,7 +16,7 @@ sourceCompatibility = 1.8
 
 group = 'org.opencadc'
 
-version = '1.3.8'
+version = '1.3.9'
```

Comment:
This will need to be 1.3.10 (.9 released yesterday)

Reply:
Updated the build number to 1.3.10.